pkg/kritis/cryptolib/jwt.go

```diff
@@ -90,11 +90,6 @@ func (v jwtVerifierImpl) verifyJwt(signature []byte, publicKey PublicKey) ([]byt
 	if err != nil {
 		return nil, errors.Wrap(err, "cannot decode payload")
 	}
-	verifyDetached(parts[2], publicKey.KeyData, publicKey.SignatureAlgorithm, append(parts[0], parts[1]...))
```

Comment:
I'm confused. Why deleted?

Reply:
It didn't need to be. I thought this would break the test but I didn't notice nothing is done with the result of this call.
I will add it back.